kythe/java/com/google/devtools/kythe/platform/java/helpers/BlockAnonymousSignatureGenerator.java

```diff
@@ -179,7 +179,13 @@ public Void visitClass(ClassTree classTree, BlockAnonymousData blockData) {
     JCClassDecl classDecl = (JCClassDecl) classTree;
     if (classTree.getSimpleName().contentEquals("")) {
       StringBuilder anonymousClassSignature = new StringBuilder();
-      anonymousClassSignature.append(getBlockSignature(classDecl.sym));
+      if (classDecl.sym != null) {
+        anonymousClassSignature.append(getBlockSignature(classDecl.sym));
+      } else {
+        String name = classDecl == null ? "" : classDecl.name.toString();
+        logger.atWarning().log(
+            "BlockAnonymous class symbol was null: %s#%s", name, blockData.anonymousNumber);
```

Comment:
Should the second %s be a %d since it's a number?

Reply:
changed